### PR TITLE
Nh pathplanner far

### PR DESCRIPTION
**Pull request by noahheller**

### CODE REVIEW

File: .pathplanner/settings.json [hunk 1]
```diff
@@ -2,11 +2,13 @@
   "robotWidth": 0.88,
   "robotLength": 0.88,
   "holonomicMode": true,
-  "pathFolders": [],
+  "pathFolders": [
+    "SpeakerFront"
```
Comment (armadilloz) [inline .pathplanner/settings.json:6]: Copy to 2024?

File: src/main/java/frc/robot/constants/GameConstants.java [hunk 2]
```diff
@@ -23,6 +23,7 @@ public class GameConstants {
     public static final boolean INTAKE_DEBUG = false;
     public static final boolean DEPLOYER_DEBUG = false;
     public static final boolean AMP_DEBUG = false;
+    public static final boolean PATHPLANNER_DEBUG = true;
```
Comment (armadilloz) [inline src/main/java/frc/robot/constants/GameConstants.java]: Undo this one

**Review by armadilloz — CHANGES_REQUESTED** [review]

File: .pathplanner/settings.json (hunk 1 above)
Comment (noahheller) [reply .pathplanner/settings.json:6]: done

File: src/main/java/frc/robot/constants/GameConstants.java (hunk 2 above)
Comment (noahheller) [reply src/main/java/frc/robot/constants/GameConstants.java]: done

File: .pathplanner/settings.json (hunk 1 above)
Comment (noahheller) [reply .pathplanner/settings.json:6]: done

**Review by armadilloz — DISMISSED** [review]

**Review by armadilloz — DISMISSED** [review]

**Review by armadilloz — APPROVED** [review]

**Pull request merged**